vault/changelog.release.2021-12-14.md

```diff
@@ -0,0 +1,85 @@
+---
+id: fRAyMtRmwkCwdRBj24eb0
+title: '0.73'
+desc: ''
+updated: 1639507560034
+created: 1639501218244
+---
+
+Dendron 0.73 has sprouted  🌱
+
+**Discover Internal Broken Links:** Want to hunt down broken, internal links in your notes? Try out the new `findBrokenLinks` action for **Dendron Doctor** and you can diagnose your problems!
+
+- `Ctrl+Shift+P` / `Cmd+Shift+P` -> `Dendron: Doctor` -> `findBrokenLinks`
+
+**Note Traits:** Dendron's new trait system allows you to create custom behavior and apply it to certain notes. This means you can have special actions take place at note creation, like automated updates to the name and `title`. Jump into the [[note traits quickstart|dendron://dendron.dendron-site/dendron.topic.traits.quickstart]] to take it for a test drive.
+
+**Navigate to Non-Note Files:** `Goto Note` now supports the ability to jump to non-note files, such as code files in your project, from wikilinks! This includes wikilinks inside of code blocks.
+
+**Rename Notes from Wikilinks:** Dendron now makes it easy for you to rename a note by placing your cursor inside a wikilink, and pressing `F2` (the `Rename Symbol` command).
+
+### Highlights
+
+- feat(commands): find broken links ([[docs|dendron://dendron.dendron-site/dendron.ref.commands#findbrokenlinks]])
+- feat(notes): Note Trait System Prototype (Phase 1) ([[docs|dendron://dendron.dendron-site/dendron.topic.traits.quickstart]])
+- feat(navigation): implement goto definition for non-note files
```

Comment:
we should add documentation to this. in [[Go to|dendron://dendron.dendron-site/dendron.topic.navigation#go-to]]

```
### Go to Definition

Dendron lets you rename symbols using the IDE's native `Go to Definition` command.

Dendron currently supports the following symbols:
- [[wikilinks|...]]
- [[NonNote Links|...]]
```